packages/graphviz.rb

```diff
@@ -22,12 +30,23 @@ class Graphviz < Package
   depends_on 'expat'
 
   def self.build
-    system "./autogen.sh"
-    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} LDFLAGS='-L#{CREW_LIB_PREFIX}' --with-libgd=yes --with-gdincludedir=#{CREW_PREFIX}/include --with-gdlibdir=#{CREW_LIB_PREFIX} --enable-tcl=no --enable-lua=no --enable-perl=no --enable-io=no --with-libexpat=yes"
-    system "make"
+    system './autogen.sh'
+    system './configure',
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}",
+           "LDFLAGS='-L#{CREW_LIB_PREFIX}'",
```

Comment:
I didn't have a problem compiling all the binaries.  Does the binary install work for you?

Reply:
No problem with the install itself but `dot -c` returns
```
crew install graphviz
graphviz: Graphviz is open source graph visualization software.
https://www.graphviz.org/
version 2.44.0
Precompiled binary available, downloading...
############################################################################################################################################################################################################# 100.0%
Archive downloaded
Unpacking archive using 'tar', this may take a while...
Installing...
Performing pre-install...
Performing post-install...
Graphviz installed!
chronos@localhost /usr/local/lib/crew/packages (master) $ dot -c
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
Warning: Could not load "/usr/local/lib64/graphviz/libgvplugin_gd.so.6" - file not found
```
I checked the syntax used in other packages. This occurs in `python27`, `python3`, `mtr`. All these have the `"LDFLAGS=-L#{CREW_LIB_PREFIX}"` syntax. Don't know why this crashes my build : I don't master ruby to investigate deeper. It really seems I need to spend some time learning Ruby ;-)

Reply:
Yeah, that is a problem.  Looks like we need to enable more options to get dot working.

Reply:
What about creating a binary for x86_64 I could download and test, using the given LDFLAGS syntax. If the compilation works here with this syntax, I strongly expect a binary produced the same way to behave normally

Reply:
It seems to me maybe this is what we need?
```
"LDFLAGS=-L#{CREW_LIB_PREFIX} -L#{CREW_LIB_PREFIX}/graphviz"
```

Reply:
It works here while compiling it. Could you generate binaries to check if these compilation options solve the error raised above ?